### PR TITLE
Add exception handling to modify_json.py

### PR DESCRIPTION
**Pull request by GDMarkou**
Add exceptions to modify_json.py to make it more understandable and user friendly. As a reminder modify_json.py is a script that allows the user to add or change a certain value to all json files included in a folder

### CODE REVIEW

File: scripts/modify_json.py [hunk 1]
```diff
@@ -68,23 +68,44 @@ def modify_json(input_folder, output_folder, variable_name, variable_value):
...
-        jfile_path = os.path.join(input_folder, jfile)
-        jdict = json_to_dict(jfile_path)
-        jdict[variable_name] = variable_value
-        save_path = os.path.join(output_folder, jfile)
-        save_dict(jdict, save_path)
+    try:
+        jsons = os.listdir(input_folder)
+    except FileNotFoundError:
+        print('Input file cannot be found or does not exist')
+        sys.exit(0)
```
Comment (bitcraft) [inline scripts/modify_json.py]: the exit code of `0` means that the script/program exited without problem.  in this case, i would think an invalid input folder is an error, so `1` would be more appropriate.  in the context of this script, its not a big deal, more of a fyi.

File: scripts/modify_json.py [hunk 2]
```diff
@@ -68,23 +68,44 @@ def modify_json(input_folder, output_folder, variable_name, variable_value):
...
+    if len(jsons) < 1:
+        try:
+            raise Exception()
+        except Exception:
+            print("Input directory is empty")
+            sys.exit(0)
```
Comment (bitcraft) [inline scripts/modify_json.py]: see above

File: scripts/modify_json.py [hunk 3]
```diff
@@ -68,23 +68,44 @@ def modify_json(input_folder, output_folder, variable_name, variable_value):
...
+        print('Folder <' + output_folder + '> was created')
 
 
 if __name__ == '__main__':
+    try:
+        if len(sys.argv) != 5:
+            raise Exception()
+    except Exception:
+        print('The number of arguments should be 4')
+        sys.exit(0)
```
Comment (bitcraft) [inline scripts/modify_json.py]: see above

File: scripts/modify_json.py [hunk 4]
```diff
@@ -68,23 +68,44 @@ def modify_json(input_folder, output_folder, variable_name, variable_value):
...
     input_folder = sys.argv[1]
     output_folder = sys.argv[2]
     variable_name = sys.argv[3]
-    variable_value = float(sys.argv[4])
+    try:
+        variable_value = float(sys.argv[4])
+    except ValueError:
+        print("Argument number 4 should be a number")
+        sys.exit(0)
```
Comment (bitcraft) [inline scripts/modify_json.py]: see above

**Review by bitcraft — DISMISSED** [review]
the exit code thing is not a big deal.  i can merge this as-is.

**Comment by GDMarkou** [comment]
No I will change it. Thank you for the review!

**Review by bitcraft — APPROVED** [review]
changes look good, thank you

**Pull request merged**